### ISSUE TITLE
Node to get a user's current equipped Avatar&Object Root

### ISSUE THREAD

**Issue by TehTurk** [Neos-Metaverse/NeosPublic#2662]
### Is your feature request related to a problem? Please describe.

Hello! I know other issues like this have been made in the past but I have a good concept of how this type of request would work.

Currently there are many objects that can essentially have the "AvatarRoot" component and makes getting a equipped Avatar's Base Slot a bit tricky. There's many ways of getting one through finding one's Head Proxy, CenterredRoot or finding a common community standard <color=red>Avatar Settings</color>

I went to double check the from issue #1990 it might require collections, but I had a decent idea.

### Relevant issues

#1990 #2557 I can't recall any others I did some searching and trouble having to find some of the previous ones.

### Describe the solution you'd like

So the running Idea I had that could work is that you can get the slot of something that is both Parented to the UserRoot and also has a Slot with an Avatar/Object Root. If it were to work say something like "Find Child by Avatar" or "Get Avatar Slot" that let you check what Avatar Root to find. That could work as it would ping the first option. If this is a bad approach apologies, I did see it required checking a collection, but I was wondering if this would be a more ok approach until collections is added.

I can understand if there are some avatars where say it's just all underneath the UserRoot but in multiple pieces. In cases like that I would think it'd good to default Avatars to say something like just a Slot that's called Avatar. The Avatar Creator currently does this with most avatars, but for a backwards compatibility maybe a temporary slot called Avatar is generated.

### Describe alternatives you've considered

Waiting for Collections.

### Additional context

This is an idea I kinda ran into due to trying to find an Avatar root for both VR and Desktop and the Default Avatar. It's too wide of a net sometimes and you won't always get an easy approach. I tried to use Head as a Child Search for both Head on just say a basically 3 quad avatar, but it ended up being unideal as the User Root already has a Head Slot.

**Comment by shiftyscales** [comment]
As you mentioned Turk, fhis is fundamentally the same issue as the aforementioned ones like #1990.

And as has been mentioned before, what's needed for the devs is understanding of the underlying issue rather fhan suggestions for how to tackle/implement it.

Requesting a specific implementation often means that the issue won't be considered/fixed if it's felt that particular approach doesn't fit well into how Neos is structured/built.

This shouldn't be its own issue because you are just looking to solve the same already known and acknowledged problem with a bandaid solution.

**Comment by TehTurk** [comment]
Ummmm, that one was a bit different hence why I elected to make a different issue as I figured discussion on this would be a little different. Originally #1990  was asking for a "save location" for avatar additions which was different. 
I was getting the actual "Object" Root of an Avatar so users avoid the tricky/extra effort way of presently trying to obtain one. 
So I'm unsure about the Duplicate tag. 

They are somewhat different problems. Prime wanted to initially leave #264 up, but was closed and wanted at least an open ticket on this , I wanted to continue that at least leave it open for others to chime in and consider.

I did label the problem properly, it's just there's not some "indepth" reason to it. Getting the Base Slot of an Avatar currently sucks. I can search for something like .fbx .gltf or do any number of ways to obtain it but that doesn't make it smooth in anyway.

That's completely fine, but it'd be nice to actually hear the consideration and see if the approach would be appropriate for right now instead of being called bandaid and closed. Collections was the answer for the previous issue in some form as it needed to enumerate all the AvatarRoots as they are components. I'm not asking for an AvatarRoot, I want something like an AvatarObjectRoot, or something like "Find by Persistent" just something that allows us to at least find it a little bit easier.
